state/block.go

```diff
@@ -3,20 +3,19 @@ package state
 import (
 	"encoding/binary"
 	"encoding/json"
-	"math/big"
-
-	"github.com/ChainSafe/gossamer/consensus/babe"
-
 	"github.com/ChainSafe/gossamer/common"
+	"github.com/ChainSafe/gossamer/consensus/babe"
 	"github.com/ChainSafe/gossamer/core/blocktree"
 	"github.com/ChainSafe/gossamer/core/types"
 	"github.com/ChainSafe/gossamer/polkadb"
+	"math/big"
 )
 
+// blockState defines fields for manipulating the state of blocks, such as BlockTree, BlockDB and Header
 type blockState struct {
-	bt          *blocktree.BlockTree
-	db          *polkadb.BlockDB
-	latestBlock types.BlockHeader
+	bt     *blocktree.BlockTree
+	db     *polkadb.BlockDB
+	header *types.Header
```

Comment:
perhaps name latestHeader? this is unclear

Reply:
sure, I still dont understand how making it latestHeader when the type is Header any more clear. But if you insist I can change.